Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/README.md

```diff
@@ -0,0 +1,126 @@
+`wildfire-report`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| md5 | MD5 hash to check. | Optional | 
+| sha256 | SHA256 hash to check. | Optional | 
+| hash | Deprecated - Use the sha256 argument instead. | Optional | 
+| verbose | Receive extended information from WildFire. Possible values are: true, false. Default is false. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| File.Name | string | Name of the file. | 
+| File.Type | string | File type, for example: "PE" | 
+| File.Size | number | Size of the file. | 
+| File.MD5 | string | MD5 hash of the file. | 
+| File.SHA1 | string | SHA1 hash of the file. | 
+| File.SHA256 | string | SHA256 hash of the file. | 
+| File.Malicious.Vendor | string | For malicious files, the vendor that made the decision. | 
+| DBotScore.Indicator | string | The indicator that was tested. | 
+| DBotScore.Type | string | The indicator type. | 
+| DBotScore.Vendor | string | Vendor used to calculate the score. | 
+| DBotScore.Score | number | The actual score. | 
+| WildFire.Report.Status | string | The status of the submissiom. | 
+| WildFire.Report.SHA256 | string | SHA256 hash of the submission. | 
+| InfoFile.EntryID | string | The EntryID of the report file. | 
+| InfoFile.Extension | string | The extension of the report file. | 
+| InfoFile.Name | string | The name of the report file. | 
+| InfoFile.Info | string | Details of the report file. | 
+| InfoFile.Size | number | The size of the report file. | 
+| InfoFile.Type | string | The report file type. | 
+| WildFire.Report.Network.UDP.IP | string | Submission related IPs, in UDP protocol. | 
+| WildFire.Report.Network.UDP.Port | string | Submission related ports, in UDP protocol. | 
+| WildFire.Report.Network.TCP.IP | string | Submission related IPs, in TCP protocol. | 
+| WildFire.Report.Network.TCP.Port | string | Submission related ports, in TCP protocol. | 
+| WildFire.Report.Network.DNS.Query | string | Submission DNS queries. | 
+| WildFire.Report.Network.DNS.Response | string | Submission DNS responses. | 
+| WildFire.Report.Evidence.md5 | string | Submission evidence MD5 hash. | 
+| WildFire.Report.Evidence.Text | string | Submission evidence text. | 
+| WildFire.Report.detection_reasons.description | string | Reason for the detection verdict. | 
+| WildFire.Report.detection_reasons.name | string | Name of the detection. | 
+| WildFire.Report.detection_reasons.type | string | Type of the detection. | 
+| WildFire.Report.detection_reasons.verdict | string | Verdict of the detection. | 
+| WildFire.Report.detection_reasons.artifacts | unknown | Artifacts of the detection reasons. | 
+| WildFire.Report.iocs | unknown | Associated IOCs. | 
+| WildFire.Report.verdict | string | The verdict of the report. | 
+
+
+#### Command Example
+```!wildfire-report hash=abcdef1234567890abcdef1234567890abcdef1234567890abcdef1234567890```
+
+#### Context Example
```

Comment:
is this really what returns from this command?
i don't understand how the PDF returns this..